app/editor/src/components/form/text/utils/formatDate.ts

```diff
@@ -0,0 +1,19 @@
+import moment from 'moment';
+
+/**
+ * Attempts to format the value in the text input.
+ * @param value The text input value.
+ * @param format The format you want to apply.  Defaults to local.
+ * @returns A formatted date value, or the original value provided.
+ */
+export const formatDate = (
```

Comment:
I don't love the location of this util.  It's really specifically related to input values, as such I want it to stay with the form values.